lib/components/admin/styled.js

```diff
@@ -161,3 +162,26 @@ export const MailablesList = styled.div`
   max-height: 120px;
   overflow-y: scroll;
 `
+
+export const MailableItem = css`
```

Comment:
I don't think this needs to be exported. Also, I think we generally don't capitalize these css variables.
```suggestion
const mailableItemCss = css`
```